infrastructure/aat.tfvars

```diff
@@ -1,2 +1,3 @@
 vault_section = "preprod"
 capacity = "2"
+external_host_name = "www.featuretoggleapi.nonprod.platform.hmcts.net"
```

Comment:
just a though, perhaps 'api' should not be part of the name?
it will be a web app for everyone using this url